docs/15-deploy-to-localstack.md

```diff
@@ -85,6 +85,15 @@ You can also write some random data to your instance by running the following co
 
 If you do not provide a number of records to ingest, then a default of 100000 is used.
 
+## Query data
+
+To query the data in your Sleeper instance, you can run the following utility script. Note that lambda queries and
+web socket queries do not work against a Sleeper instance deployed against LocalStack.
+
+```shell
+./scripts/utility/query.sh <instance-id>
```

Comment:
The rest of the documentation now starts in the deployment image, ie. in the scripts directory.